terraform.tf

```diff
@@ -1,10 +1,23 @@
 terraform {
+
+  /* Uncomment this block to use Terraform Cloud for this tutorial
+	cloud {
+			organization = "organization-name"
+			workspaces {
+				name = "learn-terraform-*"
+			}
+	}
+	*/
+
```

Comment:
The indentation of this section looks off to me - might need to remove the comment and run `terraform fmt`. Also the workspace name should be `learn-terraform-resource`, matching the tutorial.